src/database/dbConnection.js

```diff
@@ -0,0 +1,8 @@
+const { Pool } = require('pg');
```

Comment:
need to import pg module 
```
const pg = require('pg');

```

Reply:
this the same as requiring the pg and then require the pool from it!